tox.ini

```diff
@@ -1,29 +1,43 @@
 [tox]
-envlist = py{3.6,3.7,3.8}-flask{1}
+envlist = py{3.6,3.7,3.8,3.9}-flask{1,2}
```

Comment:
If we are adding support for 3.9, classifiers in `pyproject.toml` should be aware of it.

If we are not, why we need this here?